Entities/OrderEntity/Order.go

```diff
@@ -0,0 +1,13 @@
+package OrderEntity
+
+import (
+	Base "business/Entities/BaseEntity"
+)
+
+type Order struct{
+	Base.Base
+	CustomerID uint32 `binding:"required"`
```

Comment:
dont use binding. instead use validations

Reply:
Done.

But I can return my custom error(depends on validation rules) into the http response body